static/redesign/apps/gallery/templates/form-item.html

```diff
@@ -1 +1 @@
-<a href="#" class="edit-form">{{ id }}: {{ name }}</a>
\ No newline at end of file
+<p class="edit-form">{{ name }}</p>
```

Comment:
Hey Karin,

This particular change: p --> looks way better, but is no longer tab-able and there's no visual indication that it's a link. Can you switch back to "a" tag, but make it visually have more of a p-tag effect?